ui/jobs/buff_tracker.ts

```diff
@@ -604,7 +650,9 @@ export class BuffTracker {
       if ('stack' in b && b.stack !== parseInt(matches?.count ?? '0'))
         return;
 
-      this.onBigBuff(b.name, seconds, b, matches?.source);
+      this.onBigBuff(b.name, seconds, b, matches?.source, 'active');
+      // some cooldown cannot be handled by action, so also cooldown them here.
+      this.onBigBuff(b.name, seconds, b, matches?.source, 'cooldown');
```

Comment:
```suggestion
      // Some cooldowns (like potions) have no `cooldownAbility`, so also track them here.
      if (!b.cooldownAbility)
        this.onBigBuff(b.name, seconds, b, matches?.source, 'cooldown');
```

I think it's good to give an example here of which cooldowns aren't being handled.  I think also it's better for cooldowns that have abilities to not call `onBigBuff(... 'cooldown')` twice for them.  I think it technically works in that the second cooldown call will cancel the first, but I think that's a little confusing and could cause a bug in the future, so it'd be better to just call it once.  (Unless I'm misunderstanding and we need to call this even for things that have `cooldownAbility`??)

Reply:
In fact there is an issue.
Cooldown triggered by ability knows no duration. If user set a high minimum time, the will see active buff and cooldowning buff at the same time. To deal with this we should add `durationsecond` for every raidbuffs, and cannot use `useEffectduration`.

Reply:
Oh, I didn't think about that when combining them.  That makes sense!

Reply:
What happened to the conditional here? (Did I suggest something incorrect?)

Reply:
Oh seems I misunderstand you? I I thought you means that it makes sense so leave it be...

Reply:
Well, conditional added.
`durationSeconds` has been added back to prevent user with high minimum time seeing duplicated icon of active and cooldown.
But if you miss the buff, cooldown will appear after normal duration end, not at once.
(Hope that not too many user set 9999)

Reply:
Oops, sorry, I guess I'm saying that it might cause a problem in the future if buffs that have both an effect and a cooldown ability call onBigBuff('cooldown') twice.  I was trying to say that it made more sense (to me?) to only call 'cooldown' once, either on the ability or on the effect if there is no cooldown ability.  That's why I added that conditional in the suggested code.  Sorry if I didn't explain myself well enough.